packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -44,6 +44,10 @@ class BasicElement extends ReactiveElement {
   fire(name: string) {
     this.dispatchEvent(new Event(name));
   }
+
+  ice(name: string) {
```

Comment:
What's this for?

Reply:
that was me having fun with a function name, fire and ice. will change to whatever is best though

Reply:
It was to have a separate function to dispatch a custom event for a test

Reply:
removed new tests.